jobs/configure-eirini/templates/run.erb

```diff
@@ -13,23 +13,12 @@ kubectl=/var/vcap/packages/kubectl/bin/kubectl
 status "Making sure the Eirini namespace exists"
 $kubectl get namespace "<%= p("opi.kube_namespace") %>" || $kubectl create namespace "<%= p("opi.kube_namespace") %>"
 
-<% if_p("eirini.source_secrets_namespace", "eirini.source_secrets_name") do |source_secrets_namespace, source_secrets_name| %>
-
-status "Copying cc secrets to the Eirini namespace"
-secrets="$(${kubectl} get secret "<%= source_secrets_name %>" --namespace="<%= source_secrets_namespace %>" --export -o yaml | grep -E 'cc-.*|internal-ca-.*')"
-cat <<EOT >> secret.yml
----
-apiVersion: v1
-kind: Secret
-metadata:
-  name: <%= p("opi.certs_secret_name") %>
-type: Opaque
-data:
-${secrets}
-EOT
-
-$kubectl apply -f secret.yml --namespace "<%= p("opi.kube_namespace") %>"
-<% end %>
+$kubectl create secret generic "<%= p("opi.certs_secret_name") %>" -n "<%= p("opi.kube_namespace") %>" \
```

Comment:
Is `kubectl create secret` idempotent? iirc, it wasn't when we tried so we had to wrap the creation of a secret with a `kubectl apply` and call the creation with a `--dry-run` flag